picoquic/spinbit.c

```diff
@@ -34,9 +34,10 @@ void picoquic_spinbit_basic_incoming(picoquic_cnx_t * cnx, picoquic_path_t * pat
     path_x->current_spin = ph->spin ^ cnx->client_mode;
 }
 
-uint8_t picoquic_spinbit_basic_outgoing(picoquic_cnx_t * cnx)
+uint8_t picoquic_spinbit_basic_outgoing(picoquic_cnx_t * cnx,uint32_t flags)
 {
```

Comment:
I would rather not do that. How about having the logical content of these flags inside the "spin bit" context?